api-docs/rst/dev-guide/concepts.rst

```diff
@@ -13,15 +13,18 @@ How Autoscale works
 ~~~~~~~~~~~~~~~~~~~~~
 
 Rackspace Autoscale is an API-based tool that automatically scales
-resources in response to an increase or decrease in overall workload
-based on user-defined thresholds.
+resources ("scaling units", which can be cloud servers or cloud
+orchestration stacks) in response to an increase or decrease in overall
+workload based on user-defined thresholds.
 
```

Comment:
Remove parenthetical from sentence and make it a separate sentence at the end of paragraph:
  Rackspace Autoscale is an API-based service that automatically scales
 resources in response to an increase or decrease in overall
+workload based on user-defined thresholds. Resources are called *scaling units* and represent cloud servers or cloud orchestration stacks.  

Revision note:  Not sure if the above paragraph retains technical accuracy, but I think it is easier to learn/understand the Autoscale concept of resources = scaling unit concept if you remove the parenthetical.